### ISSUE TITLE
Relative Axis in line/bar Chart Visualisation

### ISSUE THREAD

**Issue by Mainer-g00t** [allegro/turnilo#552]
When displaying measures with high values it helps a lot to use relative axis to highlight differences. Is there any way to accomplish this in Turnilo?

**Comment by adrianmroz** [comment]
What is relative axis? Can you provide an example?

**Comment by Mainer-g00t** [comment]
Oops, sorry for not being more explicit. 
Let's say that you are displaying several values in the range of 1 million with differences between them in the thousands. It looks like Turnilo always sets the y-axe from 0 to the highest value. It would be good if we could choose relative y-axe starting a little bellow the lowest value

**Comment by adrianmroz** [comment]
No, we force y-scale to always contain 0 value. It is safer default. (Or maybe not if I'm thinking more about this. Turnilo users are probably people that won't get tricked by shifted y-scale ... ).

It could be an option for line chart probably. Let me convert this issue to feature request and our team will discuss feasibility of that.

**Comment by adrianmroz** [comment]
What's important, turnilo right now treats empty data point on line chart as zero value. Probably we should handle that better for non-zero y-scale minimum value.

**Comment by adrianmroz** [comment]
Visualization settings are coming: #565 maybe that would be the case for line-chart specific settings.

**Comment by luckey01** [comment]
hey team.

A bit of feedback here. We're having exactly the same issue here, where we have one measure that is a percentage, that fluctuates at 100% plus/minus 5%. I understand having zero as a default is safe, but having zero in the axis, in this case, makes it really hard to see any fluctuations.

**Comment by adrianmroz** [comment]
It is valid point but I see major problem. 

Right now turnilo can't set options per graph. If you view multiple measures on line chart you could only set hypothetical "don't include zero" for all graphs, not for only percent one. 

I don't think that such half baked feature would be usable. Probably it would just increase our tech debt.
